src/coinbase/user.ts

```diff
@@ -107,8 +81,9 @@ export class User {
    */
   public async importWallet(data: WalletData): Promise<Wallet> {
     const walletModel = await Coinbase.apiClients.wallet!.getWallet(data.walletId);
-    const addressList = await Coinbase.apiClients.address!.listAddresses(data.walletId);
-    return Wallet.init(walletModel.data, data.seed, addressList.data.data);
+    const wallet = await Wallet.init(walletModel.data, data.seed);
```

Comment:
Any reason for this not to just use `Wallet.import`?

Reply:
Are you suggesting moving these calls under new `Wallet.import` method? If so, I'm happy to make that change.

Reply:
That's currently how we do it in the Ruby SDK:

https://github.com/coinbase/coinbase-sdk-ruby/blob/1dd6a590534d5d1e0a4242b7f05c523fa857e87f/lib/coinbase/user.rb#L33-L38

https://github.com/coinbase/coinbase-sdk-ruby/blob/1dd6a590534d5d1e0a4242b7f05c523fa857e87f/lib/coinbase/wallet.rb#L32-L47

Can land that separately if needed though

Reply:
Sure, I will have a follow up PR